content/docs/guide-development.md

```diff
@@ -108,20 +136,49 @@ if you experience issues testing the service due to some "import error" or other
 
-### Developing service in Renku
+![image](https://github.com/oda-hub/hugo-odahub/blob/master/content/docs/tmp13.png)
 
+Once the project is associated to the `live-workflow` topic, it becomes visible to a bot that periodically scans the Gitlabs of the projects in the `astronomy/mmoda` domain, looking for new or modified "live" workflows that propose themselves as online services for MMODA. The bot will try to convert your notebook into a service and if this works, it will automatically add the new service  [MMODA](https://www.astro.unige.ch/mmoda) (by default, on to its staging instance). You will recieve an email similar to this:
 
-https://renkulab.io/
+![image](https://github.com/oda-hub/hugo-odahub/blob/master/content/docs/tmp14.png)
 
-TODO: explain how to run server
+You may now connect to the [MMODA](https://www.astro.unige.ch/mmoda) frontend to test the functionalities of your service, check the correctness of appearance and format of the input parameters that you have specified in the `parameters` cell of the notebook, check the formatting and correctness of the data products that are produced upon pressing the "Submit" button on the frontend etc:
+
+![image](https://github.com/oda-hub/hugo-odahub/blob/master/content/docs/tmp15_1.png)
+
+Note that some of the input parameters in the example of the Fermi/LAT [Lightcurve.ipynb](https://renkulab.io/projects/astronomy/mmoda/fermi/files/blob/Lightcurve.ipynb) notebook appear as multiple choice parameters with pre-defined values, while others are query fields. For some of the parameters, units are specified just below the query window. The names of the parameters are the names of the variables defined in the `parameters` cell of the notebook (see the screenshot of the parameters cell above in section (1) of this help page. 
+
+If the `outputs` cell of your notebook contains multiple data products, they will be shown as a list at the [MMODA](https://www.astro.unige.ch/mmoda) frontend, as shown above. The names of the list items correspond to the names of the variables defined in the `outputs` cell. Each item of the list can be previewed or downloaded by clicking on the "View" button. The preview will depend on the type of the data product that has been specified after the comment  hash `#` tag in the `outputs` cell. 
+
+You can look at different examples of the notebooks converted to services in the `astronomy/mmoda` domain on renkulab.io, to see how to format the inputs and outputs. You can also experiment with further possibilities exploring the [ontology](https://odahub.io/ontology/) of the [MMODA](https://www.astro.unige.ch/mmoda) parameters and data products. 
```

Comment:
https://renkulab.io/projects/astronomy/mmoda/mmoda-nb2workflow-example could be mentioned as a repo containing examples for all (or most of) possible output types

Reply:
```suggestion
You can explore different examples of the notebooks converted to services in the `astronomy/mmoda` domain on renkulab.io, to see how to format the inputs and outputs. If unsure, first take a look on [this simple repository](https://renkulab.io/projects/astronomy/mmoda/mmoda-nb2workflow-example). You can also experiment with further possibilities exploring the [ontology](https://odahub.io/ontology/) of the [MMODA](https://www.astro.unige.ch/mmoda) parameters and data products. 
```